tests/lib/muinstaller/go.mod

```diff
@@ -3,3 +3,24 @@ module github.com/snapcore/snapd/tests/lib/muinstaller
 go 1.18
 
 require github.com/snapcore/snapd v0.0.0-20230214100613-34da9d970bab
+
+require (
+	github.com/canonical/go-efilib v0.3.1-0.20220815143333-7e5151412e93 // indirect
```

Comment:
This seems slightly unrelated? It good to add it though :)

Reply:
Some tests were failing because muinstaller refused to compile until I did a "go mod tidy", still unsure exactly why :man_shrugging: